images/wait-for-it/config/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+variable "extra_packages" {
+  description = "The additional packages to install (e.g. wait-for-it)."
+  default     = ["wait-for-it"]
+}
+
+data "apko_config" "this" {
+  config_contents = file("${path.module}/template.apko.yaml")
+  extra_packages  = var.extra_packages
+}
+
+output "config" {
+  value = jsonencode(data.apko_config.this.config)
```

Comment:
is there a preference for inlining the config versus using a templated yaml?

Reply:
Generally I prefer the pure HCL because it enables better shared libraries and refactoring (e.g. `accts`) but this is definitely the simplest migration path.